### ISSUE TITLE
2.1.18 judging entry number style should be consistent

### ISSUE THREAD

**Issue by heneksj** [geoffhumphrey/brewcompetitiononlineentry#1114]
Version: 2.1.18

Installation URL:
https://turkeyshoot.brewcompetition.com/

Is your installation hosted on brewcompetition.com or brewcomp.com? yes

Description of Issue:
I ask the system to regenerate the judging numbers in the style-number format.  Subsequently new beers are added.  The new beers judging numbers are still in the 6-digit random number format.  I am calling this a bug, but maybe it would be an enhancement to specify which format judging numbers to use up front.  

For enhancements, please prepend "Enhancement - " to the title. For issues/bugs, please prepend the BCOE&M version number the title.

**Comment by ercousin** [comment]
How come you are generating judging numbers while people are still allowed to register entries?

Most competitions I know of wait until entry registration closes before generating judging numbers. Then if you want to add entries after you, you need to manually generate the judging number.

**Comment by heneksj** [comment]
True, but one of the sorters came late with entries.  Yeah, I fixed it manually.  Didn't notice it right away though.  So the labels were already printed.  Had to go back and re-print labels for his beer.

**Comment by ercousin** [comment]
I suppose these entries weren't registered before the entry deadline? That would explain why you needed to select a judging number.

We allow sorting volunteers to bring their entries with them, but they have to have been registered and paid by the entry deadline.

**Comment by geoffhumphrey** [comment]
When applying one of the non-default judging number schemes, the assumption is that all entries have been marked as received. The outlier here are those comps that allow staff/judges/stewards to bring their entries in after the entry window closes (as Jeff describes here). 

In a future version, most likely 3.0 planned for late next year, there will be an option for admins to choose the judging number scheme the program will utilize, thereby eliminating the "regenerate" options currently in place.
